js/src/launchAsync.ts

```diff
@@ -116,7 +116,11 @@ export function launchAsync(token: string, subdomain: string, content: Content,
             }
         };
 
-        const parent = options.parent ? options.parent : document.body;
+        options.parent = typeof(options.parent) !== 'undefined' ? options.parent : document.body;
```

Comment:
still need to check for null, should use the way you were doing it before, since that is checking for both (truthy)

Reply:
done, null and undefined are considered into parent validation